src/screens/HomeScreen.tsx

```diff
@@ -42,15 +44,8 @@ const HomeScreen: FC<Props> = () => {
         <FinderBar>
           <CategoryFilter activeCategory={filter} handleFilter={setFilter} />
         </FinderBar>
-        {error ? (
-          <ErrorBox error={error} />
-        ) : loading ? (
-          <Loader />
-        ) : filter === "none" ? (
-          <TodoList list={activeTodos} />
-        ) : (
-          <TodoList list={activeFilterTodos} />
-        )}
+        {error && <ErrorBox error={error} />}
```

Comment:
When u r using `&&`  in JSX u always should parse non-boolean values into boolean to prevent unnecessary values in DOM